collections/ansible_collections/cloudkit/config_as_code/roles/aap/vars/controller.yml

```diff
@@ -0,0 +1,216 @@
+    wait: true
+
+# Create cluster-fulfillment templates and a config-as-code template
+controller_templates:
+  - name: "{{ aap_prefix }}-create-hosted-cluster"
+    project: "{{ aap_prefix }}"
+    organization: "{{ aap_organization_name }}"
+    job_type: run
+    playbook: "playbook_cloudkit_create_hosted_cluster.yml"
+    inventory: "{{ aap_prefix }}-cluster-fulfillment"
+    execution_environment: "{{ aap_prefix }}-ee"
+    instance_groups:
+      - "{{ aap_prefix }}-cluster-fulfillment-ig"
+    allow_simultaneous: true
+    ask_variables_on_launch: true
+    verbosity: 0
+  - name: "{{ aap_prefix }}-delete-hosted-cluster"
+    project: "{{ aap_prefix }}"
+    organization: "{{ aap_organization_name }}"
+    job_type: run
+    playbook: "playbook_cloudkit_delete_hosted_cluster.yml"
+    inventory: "{{ aap_prefix }}-cluster-fulfillment"
+    execution_environment: "{{ aap_prefix }}-ee"
+    instance_groups:
+      - "{{ aap_prefix }}-cluster-fulfillment-ig"
+    allow_simultaneous: true
+    ask_variables_on_launch: true
+    verbosity: 0
+  - name: "{{ aap_prefix }}-config-as-code"
+    project: "{{ aap_prefix }}"
+    organization: "{{ aap_organization_name }}"
+    job_type: run
+    playbook: "playbook_cloudkit_config_as_code.yml"
+    inventory: "{{ aap_prefix }}-config-as-code"
+    execution_environment: "{{ aap_prefix }}-ee"
+    instance_groups:
+      - "{{ aap_prefix }}-config-as-code-ig"
+    verbosity: 0
+
+# Create cluster fulfillment and confg-as-code inventories. They are the same
+# for the moment, but spliting them will allow us to diverge easily in the
+# future.
+controller_inventories:
+  - name: "{{ aap_prefix }}-cluster-fulfillment"
+    description: "Cluster fulfilment inventory"
+    organization: "{{ aap_organization_name }}"
+  - name: "{{ aap_prefix }}-config-as-code"
+    description: "Config-as-code inventory"
+    organization: "{{ aap_organization_name }}"
+
+controller_inventory_sources:
+  - name: "{{ aap_prefix }}-cluster-fulfillment-is"
+    organization: "{{ aap_organization_name }}"
+    source: scm
+    source_project: "{{ aap_prefix }}"
+    source_path: "inventory/localhost.yml"
+    inventory: "{{ aap_prefix }}-cluster-fulfillment"
+    overwrite: true
+    overwrite_vars: true
+    update_cache_timeout: 0
+  - name: "{{ aap_prefix }}-config-as-code-is"
+    organization: "{{ aap_organization_name }}"
+    source: scm
+    source_project: "{{ aap_prefix }}"
+    source_path: "inventory/localhost.yml"
+    inventory: "{{ aap_prefix }}-config-as-code"
+    overwrite: true
+    overwrite_vars: true
+    update_cache_timeout: 0
+
+controller_schedules:
+  # Sync project every 10min to get the latest updates from the git repository
+  - name: "{{ aap_prefix }}-sync"
+    description: "Periodic sync job for {{ aap_project_name }} project"
+    organization: "{{ aap_organization_name }}"
+    unified_job_template: "{{ aap_prefix }}"
+    rrule: "DTSTART:20250331T144500Z RRULE:FREQ=MINUTELY;INTERVAL=10"
```

Comment:
Not a comment on this PR, but I am astonished to find ical syntax here :astonished: